test/functional/legacy/048_virtual_edit_spec.lua

```diff
@@ -0,0 +1,169 @@
+-- This is a test of 'virtualedit'.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, clear, execute, expect, write_file =
+  helpers.feed, helpers.insert, helpers.clear, helpers.execute,
+  helpers.expect, helpers.write_file
+
+describe('virtualedit', function()
+
+  setup(function()
+    write_file('test.in', 'a\x16b\x0dsd\n')
+  end)
+  before_each(clear)
+  teardown(function()
+    os.remove('test.in')
+  end)
+
+  it('part 1', function()
+    insert([[
+      foo, bar
+      keyword keyw
+      ]])
+    execute('set ve=all')
+    -- Insert "keyword keyw", ESC, C CTRL-N, shows "keyword ykeyword".
+    -- Repeating CTRL-N fixes it. (Mary Ellen Foster).
```

Comment:
I'll only mention this once since it's done in multiple places, but I don't think putting the comments here makes much sense. IMO, something like this:
```lua
  it('part 5', function()
    insert("'C'")
    execute('set ve=all')
    -- Make sure 'C' works.
    feed("$4lC<lt>-- should be 3 ' '<esc>")
    expect("'C'   <-- should be 3 ' '")
  end)
```
would be better off like this:
```lua
  it('part 5', function()
    -- Make sure 'C' works.
    insert("'C'")
    execute('set ve=all')
    feed("$4lC<lt>-- should be 3 ' '<esc>")
    expect("'C'   <-- should be 3 ' '")
  end)
```
I think that if the comment is mentioned before, then the reader knows what to expect, which helps avoid possible confusion by the reader. What do you think?